aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1588,6 +1594,15 @@ def get_openapi_schema(
 
         # Add routes to the OpenAPI schema
         for route in all_routes:
+
+            if route.security and not validate_openapi_security_parameters(
+                security=route.security,
+                security_schemes=security_schemes,
+            ):
+                raise SchemaValidationError(
+                    "Security configuration was not found in security_schemas or security_schema was not defined.",
```

Comment:
anything else we think it'd be useful to tell them? e.g., anything to look to fix it in the route X?

If not, leave as is.

Reply:
I think this message is very explicit. Maybe add a `See: https://docs.powertools.aws.dev/lambda/python/latest/core/event_handler/api_gateway/#security-schemes`?

Reply:
Yes, let's add that. The section itself deserves improvement too, as we can now mention that a mismatch will raise a validation error.